docs/tutorials/timeseries/forecasting-chronos.ipynb

```diff
@@ -871,30 +1186,54 @@
     "## FAQ\n",
     "\n",
     "\n",
-    "#### How accurate is Chronos?\n",
+    "#### How accurate is Chronos-2?\n",
+    "\n",
+    "Chronos-2 is the best performing (last updated: Dec 2025) time series foundation model across multiple benchmarks, including [fev-bench](https://huggingface.co/spaces/autogluon/fev-bench), [GIFT-Eval](https://huggingface.co/spaces/Salesforce/GIFT-Eval) and [Chronos Bench II](https://arxiv.org/abs/2403.07815). Details empirical results can be found in the [Chronos-2 technical report](https://arxiv.org/abs/2510.15821). The accuracy of Chronos-2 often exceeds statistical baseline models and task-specific deep learning models such as `DeepAR` and `TemporalFusionTransformer`.\n",
+    "\n",
+    "#### Does fine-tuning always improve Chronos-2's forecasting accuracy?\n",
+    "\n",
+    "Fine-tuning a foundation model like Chronos-2 involves many hyperparameter choices. AG-TS provides reasonable defaults that performed well in large-scale benchmarking, but they may not be optimal for every use case. We recommend fine-tuning only when you have a reasonable number of time series and sufficient historical data, as limited data can lead to overfitting or degraded performance. If you observe degraded accuracy, we recommend increasing the size of the training data and experimenting with different fine-tuning hyperparameters.\n",
```

Comment:
What are reasonable values here? (e.g. at least X time series / at least Y observations in total)

Reply:
That's fine